src/components/VirtualList/VirtualList.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import { Table, TableHeader, TableGridBreakpoint } from '@patternfly/react-table';
+import VirtualTable from './VirtualTable';
+import history from '../../app/History';
+import { config } from './Config';
+
+type VirtualListProps = {
+  rows: any[];
+  scrollFilters?: boolean;
+};
+
+type VirtualListState = {
+  scrollableElement: Element | null;
+  type: string;
+};
+
+const listRegex = /\/([a-z0-9-]+)(\/([a-z0-9-]+))?(\/([a-z0-9-]+))?/;
```

Comment:
Can you add a comment to tell what it's expected to match? I can see it should match something like "/istio" or "/workloads" but i'm not sure if there's more expected.

Reply:
also I'm not sure to understand what all these parenthesis are for, except the first ones.

Reply:
Yeah sorry I was thinking about to control the sort and options inside the table for sort by header feature of PF4 but I'll do later we need the UX design first